components/usage/cmd/run.go

```diff
@@ -44,12 +45,15 @@ func run() *cobra.Command {
 				log.WithError(err).Fatal("Failed to establish database connection.")
 			}
 
-			err = stripe.Authenticate(apiKeyFile)
-			if err != nil {
-				log.WithError(err).Fatal("Failed to initialize stripe client.")
+			log.Infof("payment integration with Stripe is set to %v", paymentEnabled)
+			if paymentEnabled {
+				err = stripe.Authenticate(apiKeyFile)
```

Comment:
Not strictly part of this PR but is there a way to not have the Authentication be global? This makes testing harder. 

I'd prefer if we constructed a new client with a set of credentials, and then injected that client to other components that need it.

Reply:
Yes, that's possible with the Go Stripe client. I'll address this in a follow up PR.